navigation/NavigationBarView.js

```diff
@@ -368,6 +368,15 @@ class NavigationBarView extends PureComponent {
     return null;
   }
 
+  renderBackgroundImage() {
```

Comment:
My idea was to have `renderBackground` prop and background image composer would compose `renderBackground` that renders `Image`. That way we would be able to remove `fade` styleName and pass `renderBackground` which renders gradient. That makes us closer to ability of removing react-native-linear-gradient from ui toolkit. cc @zrumenjak